source/docs/software/basic-programming/index.rst

```diff
@@ -6,3 +6,4 @@ Basic Programming
 
    git-getting-started.rst
    cpp-units
+   programming-leds
```

Comment:
This should go in the actuators section, even though it doesn't make anything move, it is an output and is very similar to the docs in that section.